### PR TITLE
Added content about society initiatives

### PR DESCRIPTION
**Pull request by TheRootOf3**
This PR introduces the following changes:
- Changed the name of `Docs` to `Our Initiatives` (including changing the directory name and rerouting the url) 
- Updated the overview page of our initatives
- Added dedicated documents for Nexus Labs and AI Foundry

### CODE REVIEW

File: docusaurus.config.js [hunk 1]
```diff
@@ -135,12 +134,20 @@ const config = {
...
-            title: 'Docs',
+            title: 'Our Initiatives',
             items: [
               {
                 label: 'Tutorial Series',
                 to: '/docs/tutorials',
               },
+              {
+                label: 'Nexus Labs',
+                to: '/docs/nexus-labs',
```
Comment (Willmish) [inline docusaurus.config.js]: Suggest renaming `docs/` folder to `our-initiatives` , and changing the corresponding paths in the docasaurus config file so the url matches the section name

**Review by Willmish — CHANGES_REQUESTED** [review]

File: docusaurus.config.js (hunk 1 above)
Comment (TheRootOf3) [reply docusaurus.config.js]: Good idea! Renamed in content files and rerouted! 🎉

**Review by Willmish — APPROVED** [review]
LGTM szef, good work 💪🏼

**Comment by TheRootOf3** [comment]
7zqhg7

**Comment by Willmish** [comment]
lmao this is precious

**Pull request merged**